### PR TITLE
Updates for 0.10.0 release

### PR DESCRIPTION
**Pull request by pameladelgado**
Updates for `0.10.0`

(deploying since it includes a bump of the ui dependency)
/deploy

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,56 @@
 .. _changelog:
 
+0.10.0
+------
+
+This release includes the new user session controller in replacement of Jupyterhub, which will incur in breaking changes (notably, all user sessions will need to be closed) when upgrading to it.
```
Comment (olevski) [inline CHANGELOG.rst:6]:
```suggestion
This release includes a new user session controller replacing Jupyterhub. The new controller is not compatible with user sessions created by Jupyterhub, therefore all user sessions need to be terminated prior to upgrading to 0.10.0.
```

**Review by olevski — CHANGES_REQUESTED** [review]
Very minor wording changes. Everything else looks good to me.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+
+Improvements
+~~~~~~~~~~~~
+
+* **Documentation**: updated documentation on proper teaching etiquette and steps to use renkulab for teaching.
+* **User sessions**: use `Amalthea <https://github.com/SwissDataScienceCenter/amalthea>`__ to control sessions through a k8s operator.
```
Comment (olevski) [inline CHANGELOG.rst:12]:
```suggestion
* **User sessions**: use `Amalthea <https://github.com/SwissDataScienceCenter/amalthea>`__ to control sessions through a k8s operator.
* **User sessions**: anonymous sessions do not require a separate namespace and renku-notebooks deployment, if enabled in the values file they now run in the same namespace as regular user sessions.
```

Comment (pameladelgado) [reply CHANGELOG.rst:12]: thanks for the suggestion, I will add this to the ``Upgrading section`` for the admins.

**Review by olevski — APPROVED** [review]

**Pull request merged**